src/backend/base/langflow/components/astra_assistants/run.py

```diff
@@ -68,36 +64,32 @@ def update_build_config(
 
     def process_inputs(self) -> Message:
         patch(OpenAI())
-        try:
-            text = ""
-
-            if self.thread_id is None:
-                thread = self.client.beta.threads.create()
-                self.thread_id = thread.id
-
-            # add the user message
-            self.client.beta.threads.messages.create(thread_id=self.thread_id, role="user", content=self.user_message)
-
-            class EventHandler(AssistantEventHandler):
-                def __init__(self):
-                    super().__init__()
-
-                def on_exception(self, exception: Exception) -> None:
-                    print(f"Exception: {exception}")
-                    raise exception
 
-            event_handler = EventHandler()
-            with self.client.beta.threads.runs.create_and_stream(
-                thread_id=self.thread_id,
-                assistant_id=self.assistant_id,
-                event_handler=event_handler,
-            ) as stream:
-                # return stream.text_deltas
-                for part in stream.text_deltas:
-                    text += part
-                    print(part)
-            return Message(text=text)
-        except Exception as e:
-            print(e)
-            msg = f"Error running assistant: {e}"
```

Comment:
I suppose this does add this prefix here, so could add it back if we think that's useful.

Reply:
This is fine, I think I added it mostly to throw a breakpoint while I was debugging.